pkg/apis/edgecore/v1alpha1/default.go

```diff
@@ -0,0 +1,171 @@
+				RegisterNodeNamespace:       constants.DefaultRegisterNodeNamespace,
+				InterfaceName:               constants.DefaultInterfaceName,
+				DevicePluginEnabled:         false,
+				GPUPluginEnabled:            false,
+				ImageGCHighThreshold:        constants.DefaultImageGCHighThreshold,
+				ImageGCLowThreshold:         constants.DefaultImageGCLowThreshold,
+				MaximumDeadContainersPerPod: constants.DefaultMaximumDeadContainersPerPod,
+				CGroupDriver:                CGroupDriverCGroupFS,
+			},
+			EdgeHub: &EdgeHub{
+				Enable:            true,
+				Heartbeat:         15,
+				ProjectID:         "e632aba927ea4ac2b575ec1603d56f10",
+				TLSCAFile:         constants.DefaultCAFile,
+				TLSCertFile:       constants.DefaultCertFile,
+				TLSPrivateKeyFile: constants.DefaultKeyFile,
+				Quic: &EdgeHubQUIC{
+					Enable:           false,
+					HandshakeTimeout: 30,
+					ReadDeadline:     15,
+					Server:           "127.0.0.1:10001",
+					WriteDeadline:    15,
+				},
+				WebSocket: &EdgeHubWebSocket{
+					Enable:           true,
+					HandshakeTimeout: 30,
+					ReadDeadline:     15,
+					Server:           "127.0.0.1:10000",
+					WriteDeadline:    15,
+				},
+			},
+			EventBus: &EventBus{
+				Enable:               true,
+				MqttQOS:              0,
+				MqttRetain:           false,
+				MqttSessionQueueSize: 100,
+				MqttServerExternal:   "tcp://127.0.0.1:1883",
+				MqttServerInternal:   "tcp://127.0.0.1:1884",
+				MqttMode:             MqttModeExternal,
+			},
+			MetaManager: &MetaManager{
+				Enable:                true,
+				ContextSendGroup:      metaconfig.GroupNameHub,
+				ContextSendModule:     metaconfig.ModuleNameEdgeHub,
+				PodStatusSyncInterval: constants.DefaultPodStatusSyncInterval,
+			},
+			ServiceBus: &ServiceBus{
+				Enable: true,
+			},
+			DeviceTwin: &DeviceTwin{
+				Enable: true,
+			},
+			DBTest: &DBTest{
+				Enable: false,
+			},
+			EdgeMesh: &EdgeMesh{
+				Enable:     true,
+				LBStrategy: LoadBalanceStrategNameRoundRobin,
+			},
+		},
+	}
+}
+
+// NewMinEdgeCoreConfig returns a common EdgeCoreConfig object
+func NewMinEdgeCoreConfig() *EdgeCoreConfig {
+	return &EdgeCoreConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       Kind,
+			APIVersion: path.Join(GroupName, APIVersion),
+		},
+		DataBase: &DataBase{
+			DriverName: DataBaseDriverName,
+			AliasName:  DataBaseAliasName,
+			DataSource: DataBaseDataSource,
+		},
+		Modules: &Modules{
+			Edged: &Edged{
+				DockerAddress:         constants.DefaultDockerAddress,
+				RuntimeType:           constants.DefaultRuntimeType,
+				NodeIP:                "",
+				ClusterDNS:            "",
+				ClusterDomain:         "",
+				RemoteRuntimeEndpoint: constants.DefaultRemoteRuntimeEndpoint,
+				RemoteImageEndpoint:   constants.DefaultRemoteImageEndpoint,
+				PodSandboxImage:       constants.DefaultPodSandboxImage,
+				HostnameOverride:      constants.DefaultHostnameOverride,
+				RegisterNodeNamespace: constants.DefaultRegisterNodeNamespace,
```

Comment:
No need to show RegisterNodeNamespace in minimal configs, we are going to deprecate RegisterNodeNamespace in a near future.

Reply:
done